CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
 
+- Implemented markdown for asset descriptions
```

Comment:
![grafik](https://github.com/sovity/edc-ui/assets/30114507/82a4841e-c0d4-42b9-9216-21d52b505a68)

this picture actually takes up the entire dialog.

- As you have used "panoramic landscape pictures", especially the first picture looks really good.
- In this case one of the pictures is even larger than the original when you click on it.
- I believe in practice those pictures will rather be:
  - Company Logo (if somebody wants to make their asset descriptions prettier or has a marketing background)
  - Some Architecture Diagram or UML diagram to explain parts of the API
  - Screenshots of some application
- I also believe in all those cases we should at least reduce the height of said images to be usable, since we have a very small dialog.
- What do you think on that?

Reply:
I thought about the pictures as well and right now they are automatically scaled to full width (since the dialog is max 1000px anyway). The height is then left variable and depends on the images aspect ratio. So theoretically one could use a very high height image and it would take up a lot of space.

When it comes to these images there are the options

- limit the max height while keeping the aspect ratio. this means images dont have uniform width
<img width="1024" alt="Screenshot 2024-01-11 at 17 03 10" src="https://github.com/sovity/edc-ui/assets/75735874/136d5094-0bf6-4eac-9b88-f8ac83fcc365">

- use full width (potentially scaling up) and variable height as now

maybe the max height approach is more reasonable but full width looks better in many cases